### DISCUSSION TITLE
Use of JuliaMono...Latin fonts

### DISCUSSION THREAD

**Discussion by cormullion** [fonsp/Pluto.jl#3025] [Pluto development] [answered]
Hi! It's been a while... I wanted to check with you about the current usage in Pluto of the JuliaMono-"Latin" fonts, ie:

- JuliaMono-RegularLatin.ttf
- JuliaMono-BoldLatin.ttf
- JuliaMono-RegularLatin.woff2
- JuliaMono-BoldLatin.woff2       

(The original idea behind making these (https://github.com/cormullion/juliamono/issues/24) was to reduce load times.)

Are you using both the `.ttf`  and the `.woff2` versions? Are you using the CSS `unicode-range` to download a subset of the complete font or a subset of the Latin font? And do you access the Latin fonts in the ZIP file `juliamono-ttf.zip` at all?

The reason for asking is to find out how the -Latin fonts are used. I've seen a number of (non-Pluto) issues with people accidentally installing the Latin fonts and wondering why they have problems. If I can streamline anything without disturbing Pluto, it would be good.

**Comment by fonsp** [comment]
Hey @cormullion, thanks for checking and for your work on JuliaMono, I love it!! Sorry for the delay, I took a small break.

We don't use the `.ttf`, only `.woff2`. You can see our CSS here: https://github.com/fonsp/Pluto.jl/blob/main/frontend/fonts/juliamono.css

We access the `.woff2` files directly from the `webfonts` folder on the github repo, not from the zip. (Specifically, we import them from https://cdn.jsdelivr.net/gh/cormullion/juliamono@0.047/webfonts/JuliaMono-Regular.woff2 etc, this uses the jsdelivr service to serve files from github as web assets, pinned to a release version. But for released Pluto versions, we download and bundle this file and include it in the Julia package, for offline availability.)

I looked into the Latin fonts, and it looks like they are not being used by the browser at all! Oopsie! I will remove the Latin imports from Pluto.

**Reply by cormullion** [reply]
Cool, thanks for looking! I might remove the -Latin.ttf fonts from the release next time - this might prevent people activating them in error. WOFFs can stay.

**Reply by fonsp** [reply, marked as answer]
Yes that sounds smart! 

We originally imported the Latin subsets to optimize page load times when only latin characters are used. But it's not working well for Pluto right now... But might be nice to get this again later! So if the Latin woff files can stay, that would be nice :)
